### PR TITLE
PLATFORM-6915 | Replace use of removed Revision.php

### PR DESCRIPTION
**Pull request by maaartyyynaa**
This branch should exist next to the master branch without merging until we are done with MW 1.37 migration

### CODE REVIEW

File: HeaderCount.hooks.php [hunk 1]
```diff
@@ -15,12 +18,13 @@ public static function renderHeadCount(Parser &$parser, $page = '', $level = '')
...
 
-        $rev = Revision::newFromTitle($title);
+		$rev = MediaWikiServices::getInstance()
+			->getRevisionLookup()->getRevisionByTitle($title);
         if ($rev === null) {
             return "'''Could not retrieve revision from $title.'''";
         }
 
-        $content = $rev->getContent(Revision::RAW);
+        $content = $rev->getContent(RevisionRecord::RAW);
```
Comment (maaartyyynaa) [inline HeaderCount.hooks.php]:
```suggestion
        $content = $rev->getContent(SlotRecord::MAIN);
```

File: HeaderCount.hooks.php [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <?php
 
+use MediaWiki\MediaWikiServices;
+use MediaWiki\Revision\RevisionRecord;
```
Comment (maaartyyynaa) [inline HeaderCount.hooks.php]:
```suggestion
use MediaWiki\Revision\SlotRecord;
```

File: HeaderCount.hooks.php [hunk 3]
```diff
@@ -15,12 +18,13 @@ public static function renderHeadCount(Parser &$parser, $page = '', $level = '')
             }
         }
 
-        $rev = Revision::newFromTitle($title);
+		$rev = MediaWikiServices::getInstance()
```
Comment (themech) [inline HeaderCount.hooks.php]: indentation is wrong

**Review by themech — APPROVED** [review]

**Review by harnash — APPROVED** [review]

**Review by maaartyyynaa — COMMENTED** [review]
can't approve but looks good to me

**Comment by maaartyyynaa** [comment]
This branch should exist next to the master branch without merging until we are done with MW 1.37 migration

**Comment by maaartyyynaa** [comment]
Closing PR, keeping branch

**Pull request closed without merging**